tests/ZfcRbacTest/Util/ServiceManagerFactory.php

```diff
@@ -59,12 +59,13 @@ public static function getApplicationConfig()
     public static function getServiceManager(array $config = null)
     {
         $config = $config ?: static::getApplicationConfig();
-        $serviceManager = new ServiceManager(
-            new ServiceManagerConfig(
-                isset($config['service_manager']) ? $config['service_manager'] : []
-            )
-        );
+        $serviceManager = new ServiceManager();
         $serviceManager->setService('ApplicationConfig', $config);
+        $serviceManagerConfig = new ServiceManagerConfig(
+            isset($config['service_manager']) ? $config['service_manager'] : []
+        );
+        $serviceManagerConfig->configureServiceManager($serviceManager);
+        $serviceManager->setAllowOverride(true);
```

Comment:
maybe better allow only in place we really need this? Not sure we even need this.